libzeth/tests/mpc/groth16/mpc_test.cpp

```diff
@@ -71,33 +71,38 @@ srs_mpc_phase2_accumulator<ppT> dummy_initial_accumulator(
 
 TEST(MPCTests, HashToG2)
 {
+    // Data here is specialised for alt_bn128. Call init_public_params on it,
+    // just in case `ppT` is set to a different curve.
+    using pp = libff::alt_bn128_pp;
+    pp::init_public_params();
```

Comment:
This is not necessary as you already init the params of `ppT` and of `alt_bn_128` in the entry point

Reply:
(covered by the above)